### PR TITLE
Feature/react login and signup forms

### PR DESCRIPTION
**Pull request by vo333ua**
## JIRA

https://ssu-jira.softserveinc.com/browse/PYTHON-208
https://ssu-jira.softserveinc.com/browse/PYTHON-207

## Code reviewers

- [ ] @meelros
- [ ] @Kardan1123
- [ ] @mizin4ik
- [x] @dimigor
- [ ] @PetrushynskyiOleksii
- [x] @Sasha1152

### Second Level Review

- [ ] @lhalam

## Summary of change

Create login and sing up forms, header 

## CHECK LIST
- [x]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: way_to_home/static/src/components/header/header.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+import StartBtn from '../startBtn/startBtn.js';
+import LoginBtn from '../loginBtn/loginBtn.js';
+import './header.css'
+
+
+class Header extends Component{
+    constructor(props){
```
Comment (Sasha1152) [inline way_to_home/static/src/components/header/header.js]: for why you use the empty constructor?

**Review by Sasha1152 — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "description": "The main idea of project – planning routes and get a notification",
   "main": "index.js",
   "dependencies": {
+    "@material-ui/icons": "^3.0.1",
```
Comment (Sasha1152) [inline package.json:8]: `"axios": "^0.18.0",` add please

File: way_to_home/static/src/components/header/header.js (hunk 1 above)
Comment (vo333ua) [reply way_to_home/static/src/components/header/header.js]: done

File: package.json (hunk 2 above)
Comment (vo333ua) [reply package.json:8]: done

File: way_to_home/static/src/components/loginForm/loginForm.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import axios from 'axios';
+import './loginForm.css';
+
+
+class LoginForm extends Component {
+
+    state = {
+        display: 'block',
```
Comment (Sasha1152) [inline way_to_home/static/src/components/loginForm/loginForm.js]: this state comes from `loginForm.css`

File: way_to_home/static/src/components/startBtn/startBtn.js [hunk 4]
```diff
@@ -1,26 +1,42 @@
 import React, {Component} from 'react';
 import Button from '@material-ui/core/Button';
+import RouteSearchForm from '../routeForm/routeForm';
 import './startBtn.css'
 
 class StartBtn extends Component{
 
-    constructor(props){
+     constructor(props){
```
Comment (Sasha1152) [inline way_to_home/static/src/components/startBtn/startBtn.js]: try to use just
```
state = {
  isRouteFormOpen: false
}
```

File: way_to_home/static/src/components/routeForm/routeForm.js [hunk 5]
```diff
@@ -77,7 +68,14 @@ class RouteSearchForm extends Component {
     this.setState({pointB: value})
   }
 
+  closeRouteResult = () =>{
+      this.setState({
+             isRouteResult: false
```
Comment (meelros) [inline way_to_home/static/src/components/routeForm/routeForm.js]: Need to change to {open: false}

File: way_to_home/static/src/components/routeResult/routeResult.js [hunk 6]
```diff
@@ -7,33 +7,37 @@ class ResultForm extends Component {
     super(props);
 
     this.state = {
+        show: true
```
Comment (meelros) [inline way_to_home/static/src/components/routeResult/routeResult.js]: Need to change to {open: true}

File: way_to_home/static/src/components/routeResult/routeResult.js [hunk 7]
```diff
@@ -7,33 +7,37 @@ class ResultForm extends Component {
...
     }
   }
 
   hideForm = () => {
-      document.getElementsByClassName('resultForm')[0].style.display = 'none'
+            this.setState({
+             show: false
```
Comment (meelros) [inline way_to_home/static/src/components/routeResult/routeResult.js]: open: false

File: way_to_home/static/src/components/routeResult/routeResult.js [hunk 8]
```diff
@@ -7,33 +7,37 @@ class ResultForm extends Component {
...
+         });
   }
 
-  
 
     render() {
-
+    const {show} = this.state
```
Comment (meelros) [inline way_to_home/static/src/components/routeResult/routeResult.js]: const {open} = this.state

File: way_to_home/static/src/components/loginForm/loginForm.js (hunk 3 above)
Comment (BoroviyOrest) [reply way_to_home/static/src/components/loginForm/loginForm.js]: removed

File: way_to_home/static/src/components/startBtn/startBtn.js (hunk 4 above)
Comment (vo333ua) [reply way_to_home/static/src/components/startBtn/startBtn.js]: i am not sure that its better. but ok.

File: way_to_home/static/src/components/routeForm/routeForm.js (hunk 5 above)
Comment (vo333ua) [reply way_to_home/static/src/components/routeForm/routeForm.js]: done

File: way_to_home/static/src/components/routeResult/routeResult.js (hunk 6 above)
Comment (vo333ua) [reply way_to_home/static/src/components/routeResult/routeResult.js]: done

File: way_to_home/static/src/components/routeResult/routeResult.js (hunk 7 above)
Comment (vo333ua) [reply way_to_home/static/src/components/routeResult/routeResult.js]: done

File: way_to_home/static/src/components/routeResult/routeResult.js (hunk 8 above)
Comment (vo333ua) [reply way_to_home/static/src/components/routeResult/routeResult.js]: done

File: way_to_home/static/src/components/startBtn/startBtn.js (hunk 4 above)
Comment (vo333ua) [reply way_to_home/static/src/components/startBtn/startBtn.js]: ok

**Review by dimigor — APPROVED** [review]

**Review by Sasha1152 — APPROVED** [review]

**Pull request merged**